webapp-django/crashstats/crashstats/templates/crashstats/crashes_per_user.html

```diff
@@ -0,0 +1,275 @@
+{% extends "crashstats_base.html" %}
+{% block site_css %}
+  {{ super() }}
```

Comment:
Sigh. We need to figure this out. 
I think I have my editor to default to 2-spaces and that's why I sometimes ruin an existing file. 
But want to know what's really sad? [THIS!](https://gist.github.com/peterbe/16f8692b31eeb88d34a1)
You can read more about it in this blog post: http://www.peterbe.com/plog/find-what-indentation-your-files-use

I think you and I need to sit down and make a decision of what we prefer. 
Then we set aside a bit of time to make some quick PRs that correct existing files and when we've done that we add a `.editorconfig` file to the root of the project. 

[In airmozilla](https://github.com/mozilla/airmozilla/blob/master/.editorconfig#L13) I use 2-spaces but it's not for any particular strong reason. Just anything being decided is nice. 

What's certainly clear is that my additions to the file shouldn't mix. If the file **was** 4-spaces, my additions should be 4-spaces too. /me corrects.